docs/src/main/sphinx/connector/hive.rst

```diff
@@ -145,30 +145,23 @@ different behaviors to the SQL version, the returned results may differ. In more
 extreme cases the queries might fail, or not even be able to be parsed and
 executed.
 
-**Experimental**
+**Advanced**
 
 The new behavior is better engineered, and has the potential to become a lot
 more powerful than the legacy implementation. It can analyze, process, and
 rewrite Hive views and contained expressions and statements.
 
-It is considered an experimental feature and continues to change with each
-release. However it is already suitable for many use cases, and usage is
-encouraged.
+You can enable the advanced behavior with ``hive translate-hive-views=true``.
 
-You can enable the experimental behavior with
-``hive.translate-hive-views=true``.
-
-Keep in mind that numerous features are not yet implemented when experimenting
-with this feature. The following is an incomplete list of **missing**
+In addition to ANSI SQL, Starburst supports the following Hive View
 functionality:
 
-* HiveQL ``current_date``, ``current_timestamp``, and others
-* Hive function calls including ``translate()``, window functions and others
-* Common table expressions and simple case expressions
-* Honor timestamp precision setting
-* Support all Hive data types and correct mapping to Trino types
-* Ability to process custom UDFs
-
+* ``UNION [DISTINCT]`` and ``UNION ALL`` in querying against Hive VIEWs
```

Comment:
We could add this in addition  .. but lets use code highlighting and lowercase otherwise consistently